scripts/generate_release_matrix.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
I dont think this is needed at all in this repo. This is something for `canfar-library`.